### ISSUE TITLE
Cannot parse wildcard export

### ISSUE THREAD

**Issue by pugnascotia** [gajus/eslint-plugin-flowtype#291]
Flow accepts syntax such as:

```javascript
export type * from './anotherModule'
```

However this causes a parse error in eslint-plugin-flow (v2.39.1).

**Comment by pnevyk** [comment]
@pugnascotia This is related to the parser. I suppose you use `babel-eslint` in which this syntax is supported since version `8.0.0` (because it uses v7 babel in which this syntax is [added](https://github.com/babel/babel/pull/5985)). If you install `8.0.0` or higher, you should be fine.

**Comment by pugnascotia** [comment]
Aha, that might explain a couple of things I've seen. I'll try it out and report back. 👍

**Comment by pugnascotia** [comment]
Upgrading the world fixed this. Thanks!
